src/textEditor.ts

```diff
@@ -34,5 +34,23 @@ export default class TextEditor {
 		
 		return vscode.window.activeTextEditor.document.lineAt(lineNo).text;
 	}
+	
+	static CopyLines(from: number, to: number): string {
+    	let register = '';
+    	for (var line = from; line <= to; line++) {
+    		register += TextEditor.ReadLine(line);
+    	}
+
+    	return register;
+    }
+    
+    static PasteBelow(text: string): Thenable<boolean> {
+    	let pos = vscode.window.activeTextEditor.selection.active;
+    	var insertPos = new vscode.Position(pos.line + 1, 0);
+    	
+    	return vscode.commands.executeCommand("editor.action.insertLineAfter").then(
+    		_ => TextEditor.Insert(text, insertPos)
```

Comment:
For fat arrow functions without any arguments, you can use `() => { my cool function }`